v2/datastream-to-spanner/terraform/samples/mysql-sharded-end-to-end/session.json

```diff
@@ -0,0 +1,7 @@
+{
+  "SessionName": "NewSession",
+  "DatabaseType": "mysql",
+  "Dialect": "google_standard_sql",
+  "SpSchema": {},
```

Comment:
It would help to give a more detailed session structure here, or refer to one in the documentation where the parameters are explained.

Reply:
Which one are you referring to? Can you give me the link?

Can't add comments to the JSON so the current one is just a placeholder that the user will need to replace.